localization/ekf_localizer/include/ekf_localizer/ekf_localizer.hpp

```diff
@@ -64,46 +64,46 @@ class Simple1DFilter
   {
     initialized_ = false;
     x_ = 0;
-    stddev_ = 1e9;
-    proc_stddev_x_c_ = 0.0;
+    dev_ = 1e9;
+    proc_dev_x_c_ = 0.0;
     return;
   };
-  void init(const double init_obs, const double obs_stddev, const rclcpp::Time time)
+  void init(const double init_obs, const double obs_dev, const rclcpp::Time time)
   {
     x_ = init_obs;
-    stddev_ = obs_stddev;
+    dev_ = obs_dev;
     latest_time_ = time;
     initialized_ = true;
     return;
   };
-  void update(const double obs, const double obs_stddev, const rclcpp::Time time)
+  void update(const double obs, const double obs_dev, const rclcpp::Time time)
   {
     if (!initialized_) {
-      init(obs, obs_stddev, time);
+      init(obs, obs_dev, time);
       return;
     }
 
     // Prediction step (current stddev_)
     double dt = (time - latest_time_).seconds();
-    double proc_stddev_x_d = proc_stddev_x_c_ * dt;
-    stddev_ = std::sqrt(stddev_ * stddev_ + proc_stddev_x_d * proc_stddev_x_d);
+    double proc_dev_x_d = proc_dev_x_c_ * dt;
```

Comment:
Please fix here in order not to change how this prediction step works for now.
```suggestion
    double proc_dev_x_d = proc_dev_x_c_ * dt * dt;
```